CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the z/OS FTP Plug-in for Zowe CLI will be documented in this file.
 
+## Recent Changes
+
+## `1.1.1` 
```

Comment:
We can work around this for automation and publishing purposes.
Please keep in mind the contribution guidelines.
https://github.com/zowe/zowe-cli-ftp-plugin/blob/master/CONTRIBUTING.md#changelog-update-guidelines

If you feel like they should change, please feel free to propose any changes you'd like to see. 😋

Reply:
Sorry for that. I misunderstand the Recent Changes and the version change. Now I got it. Thank you! 😊